components/storage/blockdevice/COMPONENT_SPIF/SPIFBlockDevice.cpp

```diff
@@ -778,3 +749,32 @@ int SPIFBlockDevice::_set_write_enable()
     } while (false);
     return status;
 }
+
+int SPIFBlockDevice::_handle_vendor_quirks()
```

Comment:
Is this a new externally accessible interface function ?

Reply:
No, it's actually private even when it might look like it's public. _private_-label has been used twice in this file.